test/e2e/persistent_volumes.go

```diff
@@ -374,17 +400,22 @@ func testPodSuccessOrFail(c clientset.Interface, ns string, pod *api.Pod) {
 
 // Delete the passed in pod.
 func deletePod(f *framework.Framework, c clientset.Interface, ns string, pod *api.Pod) {
+	if c != nil {
+		if pod != nil && len(pod.Name) > 0 {
+			framework.Logf("Deleting pod %v", pod.Name)
+			_, err := c.Core().Pods(ns).Get(pod.Name)
```

Comment:
@jsafrane I understand your comments about saving the Get api call. However, do you see any value for a test to not entirely trust the returned error and therefore to validate that the deleted resource was deleted? Maybe the answer is extra Get test is for unit testing, not e2e?